tests/main/interfaces-removable-media/task.yaml

```diff
@@ -0,0 +1,50 @@
+summary: Ensure that the removable-media interface works.
+
+details: |
+    The removable-media interface allows to access to removable storage filesystems.
+environment:
+    TESTDIR: '/media/testdir'
+    TESTFILE: '/media/testdir/testfile'
+    CONNECTED_PATTERN: ':removable-media +test-snapd-removable-media'
+    DISCONNECTED_PATTERN: '\- +test-snapd-removable-media:removable-media'
+
+prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local test-snapd-removable-media
+
+    mkdir -p "$TESTDIR"
+    touch "$TESTFILE"
+
+restore: |
+    rm -f call.error
+    rm -rf "$TESTDIR"
+
+execute: |
+    echo "The interface is not connected by default"
+    snap interfaces | MATCH "$DISCONNECTED_PATTERN"
+
+    echo "When the interface is connected"
+    snap connect test-snapd-removable-media:removable-media
+    snap interfaces | MATCH "$CONNECTED_PATTERN"
+
+    echo "Then the snap is able to read inside /run"
+    test-snapd-removable-media.sh -c "test -d /run/snapd"
```

Comment:
The interface doesn't talk about `/run/snapd` in any way. Did you find this documented as such somewhere?

The only thing that the interface grants is `/media` or alternatively `/run/media` for read access (to enumerate) and `/media/*/` or alternatively `/run/media/*/` for read-write access (to get the actual data).

Reply:
I think it also grants read on `/run`.